app/locales/es/translations.js

```diff
@@ -1,55 +1,94 @@
 export default {
   dashboard: {
-    title: 'Que deseas hacer?'
+    title: '¿Qué deseas hacer?'
+  },
+  errors: {
+    inclusion: 'no está incluido en la lista',
+    exclusion: 'es reservado',
+    invalid: 'es invalido',
+    confirmation: '{{attribute}} no corresponde',
+    accepted: 'debe ser aceptado',
+    empty: 'no puede ser vacío',
+    blank: 'no puede ser en blanco',
+    present: 'puede ser blanco',
```

Comment:
if the same pattern aplies to the same context here, ```present``` should be ```debe estar presente``` since ```puede``` indicates "maybe"